packages/xo-web/src/common/select-objects.js

```diff
@@ -691,6 +691,20 @@ export const SelectSubject = makeSubscriptionSelect(
   { placeholder: _('selectSubjects') }
 )
 
+export const SelectUser = makeSubscriptionSelect(
+  subscriber => {
+    const unsubscribeUsers = subscribeUsers(users => {
+      subscriber({
+        xoObjects: users,
+      })
+    })
+
+    return unsubscribeUsers
+  },
+  { placeholder: _('selectCreator') }
```

Comment:
This component can be used elsewhere. Since it displays the list of `users`, the placeholder should be `select user`